libraries/triumph-cmds/arguments.md

```diff
@@ -24,4 +24,17 @@ class MyClass extends BaseCommand {
     }
 }
 ```
-It is as simple as that to have arguments for your command.
\ No newline at end of file
+It is as simple as that to have arguments for your command.
+
+## Argument Name Annotation
+The `@ArgName` annotation is used to define the name of the argument should you not want the name of the argument
+to be the same as the name of the parameter.
+```java
+@SubCommand("id")
+public void execute(Sender sender, @ArgName("username") Player target) {
+    sender.sendMessage(target.getName() + " has the id: " + target.getUniqueId());
+}
+```
+In this case, the argument will be called `username` instead of the parameter name, `target`.
+
+If you would like to use the parameter name as the argument name, you can use the `-parameters` compiler flag when compiling your plugin.
```

Comment:
Make this be the first thing, you shouldn't use the annotation unless you want to make it more customizable, this is the main way of doing it.